llvm/test/Transforms/SimplifyCFG/switch-of-powers-of-two.ll

```diff
@@ -0,0 +1,90 @@
+; RUN: opt %s -passes='simplifycfg<switch-to-lookup>' -simplifycfg-require-and-preserve-domtree=1 -S | FileCheck %s
+
+target datalayout = "e-m:e-i64:64-f80:128-n8:16:32:64-S128"
+
+; Check that the range of switch of powers of two is reduced 
+define i32 @switch_of_powers(i32 %x) {
+; CHECK-LABEL: switch_of_powers
+; CHECK-LABEL: entry
+; CHECK: %{{.*}} = call i32 @llvm.cttz.i32(i32 %x, i1 true)
+; CHECK-NOT: switch i32 %x
+
+entry:
+  switch i32 %x, label %def [
+    i32 1,  label %bb1
+    i32 8,  label %bb2
+    i32 16, label %bb3
+    i32 32, label %bb4
+    i32 64, label %bb5
+  ]
+
+
+def: unreachable
```

Comment:
Please do not use the `def` abbreviation. This will introduce an ambiguity with `define`.